src/staticanalysis/bot/static_analysis_bot/report/phabricator.py

```diff
@@ -89,23 +94,40 @@ def publish_comment(self, revision, issues, patches):
         coverage_issues = [issue for issue in issues if issue.ANALYZER == COVERAGE]
 
         # First publish inlines as drafts
-        inlines = list(filter(None, [
-            self.comment_inline(revision, issue, existing_comments)
-            for issue in issues
-            if issue.ANALYZER not in ANALYZERS_WITHOUT_INLINES
-        ]))
-        if not inlines and not patches and not coverage_issues:
+        inlines_published = []
+        inlines_extras = []
+        for issue in issues:
+            if issue.ANALYZER in ANALYZERS_WITHOUT_INLINES:
+                continue
+
+            try:
```

Comment:
Instead of trying to post the comment and catching the error if we can't, couldn't we check if the issue is in a file which is in the revision?